app/models/noise.rb

```diff
@@ -149,13 +139,17 @@ def media_urls
   end
 
   def local_media_urls
-    @local_media_urls ||= media_entities.collect { |m| m.try(:[], 'media_url_https') ||  m.try(:[], 'media_url_http') }
+    @local_media_urls ||= media_entities.collect do |m|
+      m.try(:[], 'media_url_https') ||  m.try(:[], 'media_url_http')
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.